src/edu/csus/ecs/pc2/ui/SubmitSubmissionsPane.java

```diff
@@ -20,15 +25,15 @@
 import edu.csus.ecs.pc2.core.IInternalController;
 import edu.csus.ecs.pc2.core.model.IInternalContest;
 import edu.csus.ecs.pc2.ui.team.QuickSubmitter;
-import java.awt.BorderLayout;
-import javax.swing.JCheckBox;
 
 /**
  * A UI that to submit files found in a CDP.
  * 
  * 
  * @author Douglas A. Lane, PC^2 Team, pc2@ecs.csus.edu
+ * @deprecated Use {@link SubmitSampleRunsPane}
```

Comment:
It bothers me that this class is marked `Deprecated`, and yet there is still an instance of it being used in the system (specifically, in `DevelopmentPane`).  It seems to me that if a class is being replaced, it would be better to use the new class everywhere -- otherwise, we're liable to get into code inconsistencies.
However, I'm going to choose to ignore this (also, for expediency in getting this PR moved forward, I'm also ignoring -- not reviewing -- the deprecated class).